packages/components/src/components/link/_link.scss

```diff
@@ -47,7 +47,7 @@
     }
 
     &:visited {
-      color: $link-01;
+      color: $visited-link;
```

Comment:
I may be confused with this PR, but this change seems to imply we are no longer allowing teams to opt-out of having visited styles? This seems like it could cause unintended style issues in products with links that are navigated multiple times.

I believe the original intent was to provide `visited` styles but let teams allow them based on the `visited` prop, not enable them by default? 


https://github.com/carbon-design-system/carbon/pull/5239#issuecomment-580910038

https://github.com/carbon-design-system/carbon/pull/5239#issuecomment-581636154

Reply:
> this change seems to imply we are no longer allowing teams to opt-out of having visited styles

this PR allows teams to opt out of visited styles. can you explain where the implication is coming from?

Reply:
originally, visited styles were overridden which meant that teams did not have visited styles by default (referenced issues above) so visited styles were opt-in. but support for opt in visited styles was also just partial support due to the added requirement for a CSS class on top of the `:visited` pseudoclass

Reply:
This PR is setting all `visited` links to purple. When I go to the `Link` story and enter an address I've been to, I am seeing `visited` styles. How would a team, that does not want visited styles, opt-out from receiving visited styles? 

<img width="594" alt="Screen Shot 2021-03-29 at 5 27 36 PM" src="https://user-images.githubusercontent.com/11928039/112905194-09ef6c00-90b8-11eb-895d-22122a56296d.png">

Reply:
to opt out of visited styles teams can target the `:visited` pseudoclass as it was intended. I think opt-in visited styles were flawed to begin with because there is no way to know whether or not a user has visited a link due to privacy issues. for further clarification there is also the attached design spec at the top of the thread

I'm also not sure there's any large impact to returning to following the HTML spec. what kind of impact are you anticipating?

Reply:
My main concern is just that when product teams pull in this update, all visited links will be purple until they set their own `:visited` styles

Reply:
to me that change doesn't seem different from routine color contrast fixes or token updates, and the behavior would also fall in line with established usability patterns. I see the case for removing visited styles for use cases like the Gatsby theme or in brand/marketing but for the core Carbon component I think it would be a usability antipattern

Reply:
> to me that change doesn't seem different from routine color contrast fixes or token updates

Not really, tokens aren't opt-in / out. There is just one value. If Cloud has guidance that visited links are the same color as normal links and Cloud PAL pulled in this update, they would then need to ensure they set the correct visited color. Based on Laurens's response in slack, this PR doesn't seem like it's needed. 

<img width="300" alt="Screen Shot 2021-03-30 at 10 21 54 AM" src="https://user-images.githubusercontent.com/11928039/113004636-c17c9080-9141-11eb-9ffb-97b769351ae6.png">

[link](https://ibm-studios.slack.com/archives/GD8JG6JR3/p1617059068007400?thread_ts=1617058490.004700&cid=GD8JG6JR3)

We would also need to update guidance on the website if we change the default behavior 

<img width="443" alt="Screen Shot 2021-03-30 at 10 23 07 AM" src="https://user-images.githubusercontent.com/11928039/113004869-ee30a800-9141-11eb-99aa-3b8f05a1e85b.png">

Happy to create a new issue that looks into the UX of this component and so we can get feedback from product teams about link behavior, but none of these linked issues are advocating that we should set visited styles on components by default.

Reply:
to clarify this PR was opened based on contradictory information which I received from design last week together with the spec image. although they have retracted that feedback and are now aligned I think the usability issues have been brought to light again (which is separate from whether or not downstream users need to update based on Carbon changes since that would be an expected consequence from reverting antipatterns)

that being said, the guidance would still remain the same, the Gatsby theme would remain unaffected, and the difference lies in what's exported in a themed package vs what's exported in the core component library. even though this issue has been brought up previously it isn't necessarily a hill I'm willing to die on but the lack of usability input on this subject is surprising esp since it directly led to an initial misdiagnosis of #8158